### PR TITLE
ToolCard and Liked components added

### PR DESCRIPTION
**Pull request by sevtechcodes**

### CODE REVIEW

**Review by Elsass1 — APPROVED** [review]

File: app/components/ToolCard.tsx
Comment (Wyna-7) [inline app/components/ToolCard.tsx:1]: The indentation in this file looks wrong to me, not sure if it is GitHub. Can you double check?

File: app/components/ToolCard.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        />
+      </div>
+
+
+			<div className="grid grid-cols-2 gap-20">
+					<div className="mt-4 text-center flex flex-col items-left">
+						<p className="text-lg font-semibold">{tool.name}</p>
+						<p className="text-gray-600">{tool.location}</p>
+						<h2 className="text-gray-600">{tool.ownerId}</h2>
+						{/* <Avatar className="mt-2">
```
Comment (Wyna-7) [inline app/components/ToolCard.tsx:34]: If we don't need this code anymore, remove the comment

File: app/components/image.png
Comment (Wyna-7) [inline app/components/image.png:1]: Even if just for testing/hardcoding purposes, we should move this to an assests folder or something similar

File: app/explore/page copy.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        { id: '3', name: 'Drill', description: 'A tool used for drilling holes' },
+    ];
+
+    return (
+        <div>
+            <h1>Available Tools</h1>
+            <ul>
+                {tools.map(tool => (
+                    <li key={tool.id}>
+                        {/* <Link href={`/tools/${tool.id}`}> */}
```
Comment (Wyna-7) [inline app/explore/page copy.tsx:24]: Remove commented out code if not needed

File: app/explore/page.tsx
Comment (Wyna-7) [inline app/explore/page.tsx:1]: The indentation looks weird for this file too :/

File: app/globals.css [hunk 3]
```diff
@@ -1,29 +1,86 @@
...
+		:root {
+			--white: rgba(255, 255, 255);
+			--black: rgba(0, 0, 0);
+			--gray: rgba(128, 128, 128);
+			--light-green: rgba(24, 156, 140);
+			--dark-green: rgba(16, 110, 96);
+			--chat-blue: rgba(16, 77, 110);
+			--chat-green: rgba(16, 110, 49);
+		}
+    /* :root {
```
Comment (Wyna-7) [inline app/globals.css:16]: If we do not need this block, we should remove it

File: package.json [hunk 4]
```diff
@@ -12,24 +12,30 @@
   "dependencies": {
     "@heroicons/react": "^2.1.4",
     "@prisma/client": "^5.15.1",
+    "@radix-ui/react-avatar": "^1.1.0",
```
Comment (Wyna-7) [inline package.json:15]: will we have clashed if we use radix and shadcn in  the same project? let's keep an eye out for weird stuff

**Review by Wyna-7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**